easybuild/easyconfigs/f/FFTW/FFTW-3.3.4-intel-2015b-PFFT-20150905.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'ConfigureMake'
+
+name = 'FFTW'
+version = '3.3.4'
+versionsuffix = '-PFFT-20150905'
+
+homepage = 'http://www.fftw.org'
+description = """FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)
+ in one or more dimensions, of arbitrary input size, and of both real and complex data."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = [
+source_urls = [homepage]
+
+patches = ['%(name)s_%(version)s%s.patch' % versionsuffix]
```

Comment:
we have `%(versionsuffix)s` too, and if you use `"..." % foo`, you need to escape `%(name)s` like `%%(name)s`

so, use this:

```python
patches = ['%(name)s_%(version)s%(versionsuffix)s.patch']
```